physics/mock_ephemeris.hpp

```diff
@@ -0,0 +1,47 @@
+﻿#pragma once
+
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "physics/ephemeris.hpp"
+
+namespace principia {
+namespace physics {
+
+template<typename Frame>
+class MockEphemeris : public Ephemeris<Frame> {
+ public:
+  MockEphemeris() : Ephemeris() {}
+
+  MOCK_CONST_METHOD0_T(bodies, std::vector<MassiveBody const*> const&());
+  MOCK_CONST_METHOD1_T(
+      trajectory,
```

Comment:
Move to the previous line and reindent the 2nd parameter, it seems that it would fit.